controllers/heatapi_controller.go

```diff
@@ -497,8 +497,7 @@ func (r *HeatAPIReconciler) reconcileNormal(ctx context.Context, instance *heatv
 	//
 
 	serviceLabels := map[string]string{
-		common.AppSelector:     heat.ServiceName,
-		heat.ComponentSelector: heat.APIComponent,
+		common.AppSelector: fmt.Sprintf("%s-%s", heat.ServiceName, heat.APIComponent),
```

Comment:
It's quite handy to be able to do this though:
```
❯ oc get po -l service=heat
NAME                           READY   STATUS    RESTARTS   AGE
heat-api-7f775b64d-gg8gl       1/1     Running   0          10d
heat-cfnapi-5c4c8b96f7-gg6x4   1/1     Running   0          10d
heat-engine-668cd74594-k9dgk   1/1     Running   0          9d
```

By implementing this, we would no longer be able to group pods under the broader `service=heat` label and instead only be able to look up replicas of `heat-api`.